src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/VisibleMemberTable.java

```diff
@@ -454,19 +455,16 @@ private void computeVisibleFieldsAndInnerClasses(LocalMemberTable lmt, Kind kind
 
         // Filter out members in the inherited list that are hidden
         // by this type or should not be inherited at all.
-        List<Element> list = result.stream()
-                .filter(e -> allowInheritedMembers(e, kind, lmt))
-                .collect(Collectors.toList());
-
-        // Prefix local results first
```

Comment:
1. Could you incorporate this comment instead of deleting it?
2. Consider refactoring L502:L516 in a similar fashion, i.e. `a.addAll(0, b) -> Stream.concat(b, a)`.

Reply:
Done!